src/compounds/product-table/src/components/data-value.tsx

```diff
@@ -0,0 +1,33 @@
+/** @jsx jsx */
+import * as React from 'react'
+import { jsx } from 'theme-ui'
+
+import { AddonContext, CellContext, numberFormatter } from '../generics'
+
+export interface DataValueProps extends React.HTMLAttributes<HTMLSpanElement> {
+  value: number
+  unit: string
+}
+// This isn't complete yet
+const ProductTableDataValue: React.FC<DataValueProps> = ({ value, unit }) => {
+  const { inSplit } = React.useContext(CellContext)
+  const { inAddon } = React.useContext(AddonContext)
+
+  const isRow = inSplit || inAddon
+
+  const baseStyles = {
+    variant: 'productTable.dataValue.base'
+  }
+  const isRowStyles = {
+    fontSize: 'xxxl',
+    variant: 'productTable.dataValue.variants.isRow'
+  }
+
+  return (
+    <span sx={isRow ? baseStyles : isRowStyles}>
+      {numberFormatter(value, unit)}
```

Comment:
As `unit` is not required, can we add a default value of empty string? and run `.trim()` on the result? This might be better placed in the `numberFormatter` function

Reply:
The `numberFormatter` function isn't complete - I had just started on it when you said it isn't needed for now. Have left it in so that it can be built on in the future, though.